feeds/npm/npm_test.go

```diff
@@ -40,11 +41,41 @@ func TestNpmLatest(t *testing.T) {
 	if pkgs[1].Name != "BarPackage" {
 		t.Errorf("Unexpected package `%s` found in place of expected `BarPackage`", pkgs[1].Name)
 	}
-	if pkgs[0].Version != "1.0.0" {
-		t.Errorf("Unexpected version `%s` found in place of expected `1.0.0`", pkgs[0].Version)
+	if pkgs[2].Name != "BazPackage" {
+		t.Errorf("Unexpected package `%s` found in place of expected `BazPackage`", pkgs[2].Name)
 	}
-	if pkgs[1].Version != "0.4.0" {
-		t.Errorf("Unexpected version `%s` found in place of expected `0.4.0`", pkgs[1].Version)
+	if pkgs[0].Version != "1.0.1" {
+		t.Errorf("Unexpected version `%s` found in place of expected `1.0.1`", pkgs[0].Version)
+	}
+	if pkgs[1].Version != "0.5.0-alpha" {
+		t.Errorf("Unexpected version `%s` found in place of expected `0.5.0-alpha`", pkgs[1].Version)
+	}
+	if pkgs[2].Version != "1.0.0" {
+		t.Errorf("Unexpected version `%s` found in place of expected `1.0.0`", pkgs[2].Version)
+	}
+
+	fooTime, err := time.Parse(time.RFC3339, "2021-05-11T18:32:01.000Z")
+	if err != nil {
+		t.Errorf("time.Parse returned error: %v", err)
+	}
+	if !pkgs[0].CreatedDate.Equal(fooTime) {
+		t.Errorf("Unexpected created date `%s` found in place of expected `2021-05-11T18:32:01.000Z`", pkgs[0].CreatedDate)
+	}
+
+	barTime, err := time.Parse(time.RFC3339, "2021-05-11T17:23:02.000Z")
+	if err != nil {
+		t.Errorf("time.Parse returned error: %v", err)
+	}
+	if !pkgs[1].CreatedDate.Equal(barTime) {
+		t.Errorf("Unexpected created date `%s` found in place of expected `2021-05-11T17:23:02.000Z`", pkgs[1].CreatedDate)
+	}
+
+	bazTime, err := time.Parse(time.RFC3339, "2021-05-11T14:18:32.000Z")
+	if err != nil {
+		t.Errorf("time.Parse returned error: %v", err)
```

Comment:
This should be Fatalf as we cannot continue if this fails. There are a few other cases of this in this test.